components/Modal/index.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import ReactDOM from "react-dom";
```

Comment:
We could destructure here [(see another example in react docs)](https://react.dev/reference/react-dom/createPortal#reference)

```suggestion
import { createPortal } from "react-dom";
```